mvp/resources/README

```diff
@@ -1 +1,71 @@
+    - A docs.jar file that enables you to run the Galasa website locally on your machine                                                                
+    - A Notices file                                                                                                                                    
+    - Various language files                                                                                                                            
+                                                                                                 
+                                                                                                                                                    
+You can find out more about Galasa on the Galasa website https://galasa.dev, or you can host the website locally by 
+running the `docs.jar` file that is contained in this zip file on your machine or on an internal server.                                                                
+                                                                                                                                                 
+                                                                                                                                                    
+Running the Galasa website locally                                                                                                                    
+-----------------------------------                                                                                                                         
+                                     
+You can run the `docs.jar` file locally by using the standard java command with the -jar option. 
+
+Running the website on your local machine
+-------------------------------------------
+
+From a command line, run the following command in the directory into which you extracted the `docs.jar` file: 
+
+`java -jar docs.jar`
+
+The URL to view the locally hosted documentation is returned: http://localhost:9080/
+
+
+Running the Galasa website on an internal server
+----------------------------------------------------
+ 
+To host the website on an internal server so that it can be accessed by other users, set the host 
+system properties or environment variables to bind to an externally available network interface, 
+rather than localhost. For example: 
+
+`java -Dserver.http.port=12345 -jar docs.jar` or `SERVER_HTTP_HOST=example.com java -jar docs.jar`
+
+ 
+Getting started 
+--------------------                                                                                                                 
+                                                                                                                                                   
+For information about installing the Galasa plug-in and getting started with using Galasa, go 
+to _Docs > Installing the Galasa plug-in_ documentation on the Galasa website.        
```

Comment:
Has something gone askew with the _ characters it seems to be missing something

Reply:
changed to " "